src/cirq_iqm/iqm_device.py

```diff
@@ -67,6 +50,11 @@ class IQMDevice(devices.Device):
     Adds extra functionality on top of the basic :class:`cirq.Device` class for decomposing gates,
     optimizing circuits and mapping qubits.
     """
+    QUBIT_COUNT: int = None
+    """the number of qubits in the device"""
+
+    QUBIT_NAME_PREFIX: str = 'QB'
+    """prefix for naming qubits based on their numerical index"""
```

Comment:
```suggestion
    """prefix for qubit names, followed by their numerical index"""
```

Reply:
Maybe """prefix for qubit names, to be followed by their numerical index""" ?